src/commands/fund_cellar.rs

```diff
@@ -26,6 +27,24 @@ pub struct FundCellarCmd {
     pub amount_1: f64,
 }
 
+pub fn cellar_add_params(
```

Comment:
I don't think these functions are necessary since none of the values are mutated, the caller could just declare a `CellarAddParams` directly.

Reply:
Fixed with this PR: https://github.com/PeggyJV/cellar_rebalancer_rs/pull/33/commits/508238cd7ae590059d61a6bbca992e333508e4b2